exe/hooks/rthook.py

```diff
@@ -12,3 +12,8 @@
     # See https://bugreports.qt.io/browse/QTBUG-87014
     # and https://github.com/natcap/invest/issues/384
     os.environ['QT_MAC_WANTS_LAYER'] = '1'
+
+    # Rtree will look in this directory first for libspatialindex_c.dylib.
+    # In response to issues with github mac binary builds:
+    # https://github.com/natcap/invest/issues/594
+    os.environ['SPATIALINDEX_C_LIBRARY'] = sys._MEIPASS
```

Comment:
I might add a comment that `sys._MEIPASS` is the PyInstaller bundle path just so we don't have to look it up in the future.

Reply:
Out of curiosity, where was Rtree looking before?

Reply:
Based on the Rtree order of imports, I'd guess that it was looking in CWD, which for the mac binaries is at the top level of the `.app` bundle ... but why it started failing randomly is still a mystery to me!  The usual culprits (conda-forge package builds or updated verisons) were for sure not the case this time around, so I don't really know.  In any case, positively defining where the .dylib lives sounds like a good idea to me for something like a frozen binary.

The comment about `sys._MEIPASS` is updated in d391c553

Reply:
Strange! Thanks for adding that comment.